lib/config/config.go

```diff
@@ -244,6 +244,10 @@ func (cfg *Configuration) prepare(myID protocol.DeviceID) error {
 
 	cfg.MyID = myID
 
+	if len(cfg.Options.ScheduledRates.Entries) > 1 {
+		cfg.Options.ScheduledRates.Entries = cfg.Options.ScheduledRates.Entries[1:2]
+	}
```

Comment:
Without those lines there is problem that to load a configuration, you have to create a new one and then read from XML, so this loaded configuration will have default field of ScheduledRates.Entries at the first place and loaded at the second place.